packages/notification-center-angular/package.json

```diff
@@ -1,60 +1,40 @@
 {
-  "name": "@novu/notification-center-angular",
-  "version": "0.20.0",
+  "name": "@novu/angular-workspace",
+  "version": "0.0.0",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "echo \"no tests\"",
-    "lint": "echo \"No linting configured\""
-  },
-  "module": "dist/notification-center-angular/fesm2015/novu-notification-center-angular.mjs",
-  "es2020": "dist/notification-center-angular/fesm2020/novu-notification-center-angular.mjs",
-  "esm2020": "dist/notification-center-angular/esm2020/novu-notification-center-angular.mjs",
-  "fesm2020": "dist/notification-center-angular/fesm2020/novu-notification-center-angular.mjs",
-  "fesm2015": "dist/notification-center-angular/fesm2015/novu-notification-center-angular.mjs",
-  "typings": "dist/notification-center-angular/index.d.ts",
-  "files": [
-    "dist/notification-center-angular"
-  ],
-  "exports": {
-    "./package.json": {
-      "default": "./package.json"
-    },
-    ".": {
-      "types": "./dist/notification-center-angular/index.d.ts",
-      "esm2020": "./dist/notification-center-angular/esm2020/novu-notification-center-angular.mjs",
-      "es2020": "./dist/notification-center-angular/fesm2020/novu-notification-center-angular.mjs",
-      "es2015": "./dist/notification-center-angular/fesm2015/novu-notification-center-angular.mjs",
-      "node": "./dist/notification-center-angular/fesm2015/novu-notification-center-angular.mjs",
-      "default": "./dist/notification-center-angular/fesm2020/novu-notification-center-angular.mjs"
-    }
+    "test": "ng test",
+    "publish:npm": "npm run build && cd dist/notification-center-angular && npm publish"
```

Comment:
@LetItRock FYI, this is a convenience script to use in the Publishing Workflow

Reply:
@rifont we usually run the publishing from the root of the repository using `lerna publish from-package` I wonder if it wont affect it in a bad way.

Reply:
Ok, this is exactly what I was after. Yes you're right, `lerna publish` won't work as expected for this package under this script.

I'll update the PR to ensure the existing publish workflow is not interrupted.